### ISSUE TITLE
c3 wont detect lld on Nobara (Fedora based distro)

### ISSUE THREAD

**Issue by DebadityaMalakar** [c3lang/c3c#1721]
![image](https://github.com/user-attachments/assets/b093a640-2602-44d0-a6e8-e3cbcb6860c3)

**Comment by DebadityaMalakar** [comment]
I am using dnf btw to install  the required clang llvm and llvm related packages

**Comment by lerno** [comment]
Are you installing lld and lld-dev packages as well?

**Comment by DebadityaMalakar** [comment]
Yes I installed all lld and related dependencies it would be you know good to have a direct fedora c3 package

**Comment by lerno** [comment]
If someone could maintain that it would be great.

**Comment by DebadityaMalakar** [comment]
Well I would if I could but currently I need a fix for my distro

**Comment by lerno** [comment]
You did try -DC3_LINK_DYNAMIC, right?

**Comment by DebadityaMalakar** [comment]
Yeah it did and since its pretty late here I will inform you tmrw about any progress

**Comment by DebadityaMalakar** [comment]
No Progress still dont know how to fix it

**Comment by ThomasAdam** [comment]
@DebadityaMalakar 

Hey.  Rather than taking a screenshot of the last part of the output from CMake, would you mind dumping the text output here?
